agent/pool/pool.go

```diff
@@ -158,18 +127,21 @@ func (c *Conn) markForUse() {
 // streams allowed. If TLS settings are provided outgoing connections
 // use TLS.
 type ConnPool struct {
+	// The default timeout for client RPC requests in milliseconds.
+	// Stored as an atomic uint32 value to allow for reloading.
+	// TODO: once we move to go1.19, change to atomic.Uint32.
```

Comment:
```suggestion
	// clientTimeoutMs is the default timeout for client RPC requests 
	// in milliseconds. Stored as an atomic uint32 value to allow for 
	// reloading.
	// TODO: once we move to go1.19, change to atomic.Uint32.
```

Reply:
super minor nit but go idiom is to start comments with the name of the thing you're commenting though that's not even consistent in this file so totally optional!